src/Engine/Manager/RenderManager.cpp

```diff
@@ -457,6 +457,11 @@ void RenderManager::RenderEditorEntities(World& world, bool soundSources, bool p
                 glm::quat orientation = shapeComp->entity->GetWorldOrientation();
                 glm::mat4 transformationMatrix = glm::translate(glm::toMat4(orientation), position);
                 Managers().debugDrawingManager->AddCuboid(dimensions, transformationMatrix, glm::vec3(1.0f, 1.0f, 1.0f));
+            } else if (shape.GetKind() == ::Physics::Shape::Kind::Cone) {
+                glm::vec3 position = shapeComp->entity->GetWorldPosition();
+                glm::quat orientation = shapeComp->entity->GetWorldOrientation();
+                glm::mat4 transformationMatrix = glm::translate(glm::toMat4(orientation), position);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetDirectionalLights' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)